### ISSUE TITLE
Enalbe mutildex support?

### ISSUE THREAD

**Issue by vmlinz** [shadowsocks/shadowsocks-android#288]
I found proguard is really hard to make it work without complaining, and Android has been supporting multidex from some time ago, and it contains support lib to support Android versions prior to Lollipop.

How about trying to enable multidex in the development build as experiment and be free from the endless warnings of proguard?

I've update sbt and sbt android plugin to support multidex, if you are good to it, I'd like to send a pull request for multidex support.

Br,
Nick

**Comment by madeye** [comment]
Proguard is required for Scala based Android Apps, which helps to reduce the APK size.

Currently, I don't see any reason to enable multidex for Shadowsocks. This app is really small that you can even disable progurad without introducing any problem.

**Comment by vmlinz** [comment]
Thanks for the explanation.

PS: the gradle plugin for scala current have some issues by not being able to set the zinc compiler order to JavaThenScala. And it renders the app unable to be compiled, I've reported to the plugin author. Last time I tried to remove libsuperuser from the src dir and it compiles. Once it is fixed, I'll try and send pull request again.

Thank you very much for making this app :).

Br,
Nick
